lib/dbservice_web/controllers/student_controller.ex

```diff
@@ -735,44 +735,74 @@ defmodule DbserviceWeb.StudentController do
   end
 
   def update_user_enrollment_records(conn, params) do
-    student = Users.get_student_by_student_id(params["student_id"])
-    user_id = student.user_id
-    academic_year = params["academic_year"]
+    with student when not is_nil(student) <-
```

Comment:
Are tehre some API docs for db-service? We should add that ASAP if not

Reply:
We currently have a Swagger doc for db-service, which serves as the API documentation.